packages/neuron-ui/src/components/Settings/index.tsx

```diff
@@ -1,3 +1,63 @@
 import React from 'react'
+import styled from 'styled-components'
+import { Launch } from 'grommet-icons'
 
-export default () => <h1>Settings</h1>
+export const FullView = styled.div`
+  background-color: white;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  z-index: 1;
+  box-sizing: border-box;
+`
+
+const WelcomeDiv = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 80px;
+  .buttonGroup {
+    button {
+      height: 40px;
+      width: 140px;
+      cursor: pointer;
+    }
+  }
+`
+
+export default (props: any) => (
```

Comment:
Like said above, let's make `Settings` the preferences. So the onboard (welcome) component should be an independent part.

Reply:
ok, I understand. previewly  I create the welcome component. I thought you want to place create/import function in setting from your prototype draft. I would recreate welcome component